storage-manager/src/S3Storage.cpp

```diff
@@ -361,29 +374,29 @@ int S3Storage::getObject(const string& _sourceKey, std::shared_ptr<uint8_t[]>* d
 {
   uint8_t err;
   size_t len = 0;
-  uint8_t* _data = NULL;
+  uint8_t* _data = nullptr;
   string sourceKey = prefix + _sourceKey;
 
-  ms3_st* creds = getConnection();
+  auto creds = getConnection();
```

Comment:
The variable `creds` is a result of `getConnection`. I am either stupid or the variable name is misguiding.

Reply:
Good point, thanks.